src/app/dataset-view/additional-components/overview-component/overview.component.ts

```diff
@@ -110,6 +110,18 @@ export class OverviewComponent extends BaseComponent implements OnInit {
         }
     }
 
+    public get canAddPushSource(): boolean {
+        if (this.currentState && this.datasetPermissions.permissions.canCommit) {
+            return (
```

Comment:
Place a TODO, so you don't forget to replace it to faster API

Reply:
Add TODO: comments